github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger that run job for GitHub Pull Request activities.
+    Trigger that launches builds in response to GitHub Pull Request activities.
```

Comment:
Maybe "Trigger which integrates with GitHub Pull Request activities to launch runs in response to activity on pull requests"?